auth.js

```diff
@@ -0,0 +1,32 @@
+var app = require('./app');
+var https = require('https');
+var path = require('path');
+var fs = require('fs');
+var auth = {};
+
+//To get the oauth token from slack server.
+auth.getToken = function (req, res, next) {
+
+ var authCode =  req.query.code;
+ var slackKey = JSON.parse(fs.readFileSync(path.join(__dirname + '/config/slackAppKey.json'), 'utf8'));
```

Comment:
Would like to see a `README.md` file for how to set this system up? This should include documentation on the process of registering the application with Slack, setting up the Heroku server, getting the slack keys, etc.